app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -0,0 +1,100 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      If maths is your passion, you'll know that it's not just learning formulas or solving equations. It's a way of thinking and problem solving that’s important in every walk of life.
+    </p>
+
+    <p>
+      An exciting career spent passing your knowledge on to the next generation could be for you.
+    </p>
+<p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of £28,000 or scholarships of £30,000 are available for eligible trainee maths teachers</a>.</p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Making maths add up",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+      <p>Maths is critical to science, technology and engineering as well as being fundamental for most jobs.</p>
+
+<p>A good understanding of maths has a lot of benefits for young people to prepare them for their future, like using financial maths and data handling.</p>
+<p>It's also our government's goal for all young people to study maths to age 18, equipping them with the skills they need for life.</p>
+    <% end %>
+  </section>
+  <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "Don’t you just love that feeling when you didn’t know what to do, then the answer just floods through you.",
+    name: "Child explaining her maths learning",
```

Comment:
@caitlin-reidy I'm sure I know the answer to this! But do we have a first name for the child or an age for the child or if they're primary or secondary?

Reply:
@gemmadallmandfe the NCETM gave this quote but only gave 'child explaining her maths learning' - are we able to use a fake name or maybe 'Child, aged 12'?

Reply:
I don't think we should make up a name or age

Does this quote add anything to the page - I'm not sure it does? It seems a bit odd that it is from a child rather than a teacher?

Reply:
Ok have deleted the quote!